phoenix-core/src/main/java/org/apache/phoenix/expression/BlobExpression.java

```diff
@@ -55,10 +57,16 @@
     };
     private final LobStore lobStore;
 
-    public BlobExpression(Object value, SortOrder sortOrder, Determinism determinism) {
+    public BlobExpression(Object value, SortOrder sortOrder, Determinism determinism,
```

Comment:
prefer factory pattern rather than doing the new in here.. but for hackathon this is okay

Reply:
I was being lazy :P, will change